saw-core/src/SAWCore/SCTypeCheck.hs

```diff
@@ -90,7 +92,7 @@ type TCState = Map TermIndex Term
 --
 -- * Can throw 'TCError's
```

Comment:
Make sure to update the Haddocks here to say what the purpose of the `Map LocalName (ExtCns Term)` is. Alternatively, if we factor out the tuple into its own data type, we could add Haddocks to the fields of that data type.

Reply:
Fixed in 965fe21f2965b3178a470b4ecec8b28128810ca9.